packages/components/src/components/drag-and-drop-file-container/src/index.tsx

```diff
@@ -96,14 +98,14 @@ const DragAndDropFileContainer = React.forwardRef<
               </Box>
             )}
 
-            <Text css={{ marginBottom: "$35" }}>
-              {isDragReject ? rejectText : uploadText}
-            </Text>
+            <Box css={{ marginBottom: "$35" }}>
+              <Text>{isDragReject ? rejectText : uploadText}</Text>
+            </Box>
```

Comment:
The `css` should have only been on box, as its our utility component. Not really sure how this worked before.